x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_page.tsx

```diff
@@ -142,6 +142,14 @@ export const LogRateAnalysisPage: FC<Props> = ({ stickyHistogram }) => {
     });
   }, [dataService, searchQueryLanguage, searchString]);
 
+  useEffect(
+    () => {
+      setInitialAnalysisStart(appStateToWindowParameters(stateFromUrl.wp));
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [JSON.stringify(stateFromUrl.wp)]
+  );
```

Comment:
Is it supposed to run only once, or does it sync every time the URL state is changed? Can you also omit `JSON.stringfy`? 

```suggestion
  useEffect(function syncWindowParams() {
      setInitialAnalysisStart(appStateToWindowParameters(stateFromUrl.wp));
    },
    // eslint-disable-next-line react-hooks/exhaustive-deps
    [JSON.stringify(stateFromUrl.wp)]
  );
```